GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -14,6 +14,7 @@
 
 using UnityEditor;
 using UnityEngine;
+using QuickDeployConfig = GooglePlayInstant.Editor.PlayInstantQuickDeployConfiguration;
```

Comment:
I'd rather we not need to use an alias like this. Alternatively, I think it's fine if you want to rename the class to QuickDeployConfig.